### PR TITLE
API-1374: [OnPrem] Write an article about CE installation

### PR DESCRIPTION
**Pull request by tomglvng**
- Documentation about consume worker
- Documentation about messages purge

### CODE REVIEW

File: install_pim/manual/event_api.rst [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    [Install]
+    WantedBy=multi-user.target
+
+Create ``/usr/local/bin/akeneo_events_api_consumer_launcher.sh``:
+
+.. code-block:: bash
+    :linenos:
+
+    su -c "/path/to/akeneo/bin/console messenger:consume webhook --env=prod &" akeneo
+    exit 0
```
Comment (pchasle) [inline install_pim/manual/event_api.rst:117]: We need to check if this sh file is still necessary.

File: install_pim/manual/index.rst [hunk 2]
```diff
@@ -18,3 +18,4 @@ Once the requirements are fulfilled, you are ready to install Akeneo PIM. Otherw
    installation_ce
    installation_ee_archive
    daemon_queue
+   event_api
```
Comment (pchasle) [inline install_pim/manual/index.rst:21]:
```suggestion
   events_api
```
Same for the file name

File: install_pim/manual/event_api.rst [hunk 3]
```diff
@@ -0,0 +1,219 @@
+Setting up the event api
```
Comment (pchasle) [inline install_pim/manual/event_api.rst:1]:
```suggestion
Setting up the Events API
```
Same for the file name

File: install_pim/manual/event_api.rst (hunk 1 above)
Comment (tomglvng) [reply install_pim/manual/event_api.rst]: Checked in https://github.com/akeneo/pim-docs/pull/1560/commits/c7a35fef6f0edcb52f15bc658c19d3b47fc843dc

**Review by pchasle — APPROVED** [review]

**Review by willy-ahva — APPROVED** [review]

**Pull request merged**